core/src/main/java/com/linecorp/armeria/server/ServiceBindingBuilder.java

```diff
@@ -159,6 +159,26 @@ public ServiceBindingBuilder produces(Iterable<MediaType> produceTypes) {
         return (ServiceBindingBuilder) super.produces(produceTypes);
     }
 
+    @Override
+    public ServiceBindingBuilder matchesParamPredicates(String... paramPredicates) {
```

Comment:
- How about just `matchesParams(String... expressions)` and `matchesHeaders(String... expressions)`?
- It seems like we use `CharSequence...` and `Iterable<? extends CharSequence>` in some places. Could we make it consistent in all places?

Reply:
I couldn't remember why I changed `String` to `CharSequence` for headers. BTW, it was not necessary. Will change them to `String`. Thanks!